capstone/src/main/webapp/cover.css

```diff
@@ -0,0 +1,89 @@
+/* Links */
+a, a:focus, a:hover {
+  color: #fff;
+}
+
+/* Custom default button */
+.btn-secondary, .btn-secondary:hover, .btn-secondary:focus {
+  color: #fff;
+  text-shadow: none; /* Prevent inheritance from `body` */
+  background-color: #451735;
+  border: .05rem solid #fff;
+}
+
+/* Base structure */
+html, body {
+  height: 100%;
+  background-image: url(/assets/Pattern-Randomized.svg);
+}
+
+body {
+  display: -ms-flexbox;
+  display: flex;
+  color: #451735;
+}
```

Comment:
(super) nit: Let's move these `html` and `body` styles up to the top of the file. I've found that it's helpful to write CSS rules in the roughly the same "order" in which the selectors or classes are written in the HTML.

For example:
```
html { // Some rules... }
body { // Some rules... }
.some-class { // Some rules... }
header { // Some rules... }
footer { // Some rules... }
```
Rather than something like:
```
html { // Some rules... }
header { // Some rules... }
footer { // Some rules... }
.some-class { // Some rules... }
body { // Some rules... }
```

Reply:
addressed in "restructure CSS"